src/helpers/__tests__/flare.test.ts

```diff
@@ -228,7 +236,8 @@ describe('flare', () => {
       expect(res).not.toBeUndefined()
       const [start, end] = res
       expect(start).toBe(0)
-      expect(end).toEqual(now)
+      expect(end).toBeLessThan(9999999999999)
+      expect(end).toStrictEqual(now)
```

Comment:
We could also remove line 234 (`const now = Date.now()`) and only assert that `end` is less than `9999999999999`. But returning `8888888888888` would make the assertion pass.

So I kept `now` and used fake timers instead. In real world, we don't rely on two calls of `Date.now()`.

Reply:
cc. @nhulston